File: FirstClass/problem6.m
disp ("Solution: ")
A = (pi .^2 - 3 * sqrt (pi) -exp (1))/ (4 .* tanh (2));
display (A, "A");
B = log (2 + cos (pi/8));
display (B, "B");
res = A + B;
display (res, "Respuesta")

File: FirstClass/Python/problem_6.py
import math

if __name__ == '__main__':
    A = (math.pi ** 2 - 3 * math.sqrt(math.pi) - math.exp(1)) / (4 * math.tanh(2))
    print(f'A = {A}')
    B = math.log(2 + math.cos(math.pi / 8))
    print(f'B = {B}')
    print(f'A + B = {A + B}')